README.md

```diff
@@ -497,9 +499,9 @@ jobs:
           mkdir -p $cache_dir
           tar -xzf cache-download/$cache_archive -C $cache_dir
 
-          # Unfortunately, the permissions expected within renovate's docker container
-          # are different than the ones given after the cache is restored. We have to
-          # change ownership to solve this. We also need to have correct permissions in
+          # Unfortunately, the permissions expected within Renovate's Docker container
+          # are different than the ones given after the cache is restored. We must
+          # change ownership to solve this. We also must have the correct permissions in
           # the entire /tmp/renovate tree, not just the section with the repo cache.
           sudo chown -R runneradmin:root /tmp/renovate/
           ls -R $cache_dir
```

Comment:
This seems to no longer work. On the other hand, there is now a `docker-user` option.

I am now using the following:
```yaml
      - name: Determine current user
        id: id
        run: |
          echo "user=$(id -u)" >> $GITHUB_OUTPUT
          echo "group=$(id -g)" >> $GITHUB_OUTPUT
      - uses: renovatebot/github-action@v40.3.4
        with:
          configurationFile: renovate.json5
          token: ${{ secrets.RENOVATE_TOKEN }}
          renovate-version: 37.421.3
          # used to mount cache
          docker-user: "${{ steps.id.outputs.user }}:${{ steps.id.outputs.group }}"
          docker-volumes: |
            /tmp/renovate/cache:/tmp/renovate/cache
        env:
          # This enables the cache -- if this is set, it's not necessary to add it to renovate.json.
          RENOVATE_REPOSITORY_CACHE: ${{ github.event.inputs.repoCache || 'enabled' }}
```